src/logic/BoothLogic.h

```diff
@@ -125,7 +126,9 @@ namespace selfomat {
             // really needs to print or not. if it wants to print it prints.
             // Finally it unlocks both mutex and exits
             boost::mutex cancelPrintMutex;
+            boost::mutex confirmPrintMutex;

```

Comment:
We should merge these to a single mutex (e.g. "printDecisionMutex"), since we just lock two threads from accessing the "user decision" information at the same time.

Basically:
- rename one mutex and delete the other
- everywhere you lock either of the mutexs, lock the renamed one